cfde_deriva/metrics.py

```diff
@@ -156,17 +168,11 @@ def register_datapackage_measurements(registry_catalog, submission_id, records):
     registry_catalog.put(
         '/attributegroup/CFDE:datapackage_measurement/datapackage,metric;value,numerator,denominator,comment',
         json=need_update,
-    ).json() # discard response data
-
-def get_datapackage_measurements(registry_catalog, submission_id, headers=DEFAULT_HEADERS):
```

Comment:
Please rebase your change on master so this additional headers param is not pruned out in the PR...